service/flow_helpers/account_test.go

```diff
@@ -55,3 +57,70 @@ func TestAccountCaching(t *testing.T) {
 		t.Fatal("expected accounts to not equal")
 	}
 }
+
+func TestAccountAvailableKeys(t *testing.T) {
+
```

Comment:
for running these unit tests, this is the instruction to run it: https://github.com/dapperlabs/flow-pds/pull/3/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R21 ?